server/datastore/datastore.go

```diff
@@ -8,6 +8,8 @@ import (
 * must implement these functions */
 type DB interface {
 	Init(config DatabaseConfig) error // Initialize pgx connection
+	CreateMaintenanceRequests(request string, userID int, time time.Time) (int, error)
```

Comment:
Rather than taking in each individual param, why not take in a MaintenanceRequest struct? That's simpler and makes it easier to update in the future. (I mention this below too)

Reply:
fixed: used datastore.MaintenanceRequest struct as param